challenge_2/rust/deg4uss3r/challenge_2/src/lib.rs

```diff
@@ -0,0 +1,34 @@
+use std::vec::Vec;
+
+pub fn find_lonely (mut input_vec: Vec<u64>) -> u64 {
```

Comment:
May want to make that return `Option<u64>`, just in case there isn't a lonely in your input data. Safety first!